freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -348,3 +367,53 @@ def _create_original_to_rebuilt_nvrs_map(self):
                             block_build_nvr['version'] == build_nvr['version']:
                         nvrs_mapping[block_build] = build['nvr']
         return nvrs_mapping
+
+    def _prepare_builds(self, db_event, bundles_by_digest, to_rebuild_digests):
+        """
+        Prepare models.ArtifactBuild instance for every bundle that will be
+        rebuilt
+
+        :param models.Event db_event: database event that will contain builds
+        :param dict bundles_by_digest: mapping of bundle digest to bundle data
+        :param list to_rebuild_digests: digests of bundles to rebuild
+        :return: builds that already in database and ready to be submitted to brew
+        :rtype: list
+        """
+        builds = []
+        csv_mod_url = conf.freshmaker_root_url + "/api/2/pullspec_overrides/{}"
+        for digest in to_rebuild_digests:
+            bundle = bundles_by_digest[digest]
+            # Reset context to db_event for each iteration before
+            # the ArtifactBuild is created.
+            self.set_context(db_event)
+
+            rebuild_reason = RebuildReason.DEPENDENCY.value
```

Comment:
Could you please make this `DIRECTLY_AFFECTED` instead so that BOTAS will know to file advisories for these bundle images?

Reply:
Sure. That's actually the thing I was not sure about, I was wondering if it's 'directly affected' or 'dependency'.